src/oci-limits-mcp-server/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# OCI Limits MCP Server Dependencies
```

Comment:
you can specify these dependencies in the pyproject.toml file https://github.com/oracle/mcp/blob/main/src/oci-compute-mcp-server/pyproject.toml#L12-L15 and delete this file

You'll also need a dev dependencies block: https://github.com/oracle/mcp/blob/main/src/oci-api-mcp-server/pyproject.toml#L34-L38